tests/test_enhanced.py

```diff
@@ -0,0 +1,86 @@
+import os
+
+import pydicom
+import unittest
+import hazenlib.utils as hazen_tools
+
+# class TestEnhanced(unittest.TestCase):
+def main():
+    TEST_DATA_DIR = "/Users/sophieratkai/Documents/projects/hazen_docs/ACR_se_enhanced"
+    enhanced_folder = os.path.join(TEST_DATA_DIR, 'enhanced')
+    folder = "/Users/sophieratkai/Documents/projects/hazen_docs/Philips_DicomTest/Res/Enhanced/DICOM"
```

Comment:
You have hard-coded filepaths from your local machine. Slightly puzzled why these haven't triggered as errors in the GitHub Actions. Can you check these tests are actually being run and passing, thanks.